release-notes/release-notes/5.8-release-notes.md

```diff
@@ -17,12 +17,28 @@ For instructions on how to upgrade your platform to the latest version, please r
 
 ### Platform
 
+### Updated POST endpoint for Saved Queries
+
```

Comment:
CIN-05793. Not sure if I should mention more.

Reply:
This is not in 5.8

Reply:
Removing now. Will resolve when removed.